breathecode/certificate/receivers.py

```diff
@@ -14,3 +16,10 @@ def post_save_user_specialty(sender, instance: UserSpecialty, **kwargs):
 
     elif instance._hash_was_updated and instance.status == 'PERSISTED' and not instance.preview_url:
         tasks.take_screenshot.delay(instance.id)
+
+
+@receiver(student_edu_status_updated, sender=CohortUser)
+def generate_certificate(sender, instance: CohortUser, **kwargs):
+    print(instance)
+    if instance.educational_status == 'GRADUATED':
```

Comment:
The certificate just must be autogenerated if the student coming from a saas cohort or academy cohort, if cohort.is_available_as_saas in [False] it does not take in consideration the value in the academy